POC-OOBE/OOBE-Sandbox/PowerToys Settings Sandbox/Views/onLaunchContentDialog.xaml

```diff
@@ -0,0 +1,44 @@
+﻿<ContentDialog
+    x:Class="PowerToys_Settings_Sandbox.Views.onLaunchContentDialog"
+    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+    xmlns:local="using:PowerToys_Settings_Sandbox.Views"
+    xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+    xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+    mc:Ignorable="d"
```

Comment:
Sorry, I am not super familiar with this tech stack. What does this line do?

Reply:
Those are automatically generated by xaml/visual studio. There is a long definition of their function on the [xaml documentation site](https://docs.microsoft.com/en-us/dotnet/desktop-wpf/fundamentals/xaml)